projects/plugins/jetpack/_inc/lib/class-jetpack-tweetstorm-helper.php

```diff
@@ -1649,7 +1647,7 @@ function ( $url ) use ( $validator ) {
 		$requests = array_filter( $requests );
 
 		// Remove this check once WordPress 6.2 is the minimum supported version.
-		if ( version_compare( $wp_version, '6.2-alpha', '<' ) ) {
+		if ( ! class_exists( '\WpOrg\Requests\Hooks' ) ) {
```

Comment:
Nitpick, feel free to ignore
```suggestion
		if ( ! class_exists( \WpOrg\Requests\Hooks::class ) ) {
```
Same for the other instances below.